index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
+INSTRUCTIONS:
+  - Select the project you would 
+    like to complete from the dropdown 
+    menu.
+  - Click the "RUN TESTS" button to
+    run the tests against the blank 
+    pen.
+  - Click the "TESTS" button to see 
+    the individual test cases. 
+    (should all be failing at first)
+  - Start coding! As you fulfill each
```

Comment:
Kindly delete the unused code into this file to make it clean

Reply:
- [ ] I think it would be better to give the why you give the suggestion because even though it might seem obvious to you it might not be so obvious to the person you are reviewing 😉